services/services.rst

```diff
@@ -23,36 +23,18 @@ SuMIRe/PFS PBworks wiki
 
 "SuMIRe/PFS PBworks" is the only official wiki system for PFS. 
 
-======
-Login and ToC
-======
-
 After logging in at `PBworks <http://sumire.pbworks.com/>`_, 
 you will get a project index page named 
-"Subaru Measurement of Images and Redshifts (SuMIRe)". 
-This page contains links to contents in the wiki, such as 
-
-* PFS Project Office (General links, including link to telecon indexes)
-* Documents (Link list to documents)
-* Meeting, conference, etc.  (Link list to meetings)
-* Mailing lists (List of avail lists)
-* PFS working groups (Member list)
-
-======
-Page indexes
-======
-
-For some continuous meetings, index for each agenda/memo pages are avail.
-
-* `Systems Engineering group telecon <http://sumire.pbworks.com/Systems-engineering-telecons>`_
-* `Manager group telecon <http://sumire.pbworks.com/Management-telecons>`_
-
+`"Subaru Measurement of Images and Redshifts (SuMIRe)" <https://sumire.pbworks.com/w/Subaru%20Measurement%20of%20Images%20and%20Redshifts%20%28SuMIRe%29>`_. 
+This page contains links to contents in the wiki, also 
+`SideBar <https://sumire.pbworks.com/w/SideBar>`_ is shown at right side of 
+every wiki page as list of links with short names. 
 
 ======
 Files
 ======
 
-Also, you can find all files uploaded into this SuMIRe/PFS PBworks from 
+You can find all files uploaded into this SuMIRe/PFS PBworks from 
 `folder listing <http://sumire.pbworks.com/w/browse/#view=ViewAllFiles>`_.
 Some files are categorized into *FOLDERS*, and you can get each list by 
 clicking FOLDER name at left side. 
```

Comment:
at left side -> on left side
Do we have to mention "page" here? (I guess you mean "page and files" here, though...)

Reply:
ah,, page is actually a special case of file on pbworks, so these will be listed. but pbworks just notes as 'Files', so I've left this part as files...